static/static_file_modifier_test.go

```diff
@@ -48,7 +100,7 @@ func TestStaticModifierOnRequest(t *testing.T) {
 
 	res := proxyutil.NewResponse(http.StatusOK, nil, req)
 
-	mod := NewModifier(tmpdir)
+	mod := NewModifier(tmpdir, nil)
 	if err := mod.ModifyRequest(req); err != nil {
 		t.Fatalf("ModifyRequest(): got %v, want no error", err)
 
```

Comment:
I'd prefer a few more test cases:
One where there's an explicit mapping for a file, the file exists both at that path and at the root path, asserting that you get the one from the mapping
One where there's an explicit mapping for a file, the file doesn't exist at that path or the root path, asserting the 404
One where there's an explicit mapping, the file doesn't exist there, but it does exist at the root, asserting the 404